src/ui/DynamicFacet/DynamicFacetValues/DynamicFacetValues.ts

```diff
@@ -150,6 +154,24 @@ export class DynamicFacetValues implements IDynamicFacetValues {
     }
   }
 
+  private appendSelectedCollapsedValues(fragment: DocumentFragment) {
+    if (!this.hasSelectedValues) {
```

Comment:
Same comment about using idle instead of selected

Reply:
Would be appendExcludedCollapsedValues IMO.
Don't worry, it would not be forgotten 😛